chain/client/Cargo.toml

```diff
@@ -13,8 +13,8 @@ rocksdb = "0.13"
 log = "0.4"
 rand = "0.7"
 serde = { version = "1.0", features = ["derive"] }
-serde_derive = "1.0"
 serde_json = "1.0"
+strum = { version = "0.16.0", features = ["derive"] }
```

Comment:
what does `strum` do?

Reply:
It implements a derive which adds a method returning a enum variant name as static str. I use it to "serialize" SyncStatus here: https://github.com/nearprotocol/nearcore/pull/1751/files#diff-13c136d6977c948850fed96e9b92b40cR129